### PR TITLE
[PRTL-3237] rename AWG renewal variable for CRA

### PR DESCRIPTION
**Pull request by caravinci**
## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [x] `qa-purple`

## Description of Changes
Renames the environment variable, according to the Create React App env. validation requirement.

### CODE REVIEW

File: src/packages/@ncigdc/utils/constants.js [hunk 1]
```diff
@@ -816,7 +816,7 @@ export const DEV_USER_CA = [ // controlled access mock
 
 export const AWG_TOKEN_EXPIRY = (
   localStorage.AWG_TOKEN_EXPIRY ||
```
Comment (singergr) [inline src/packages/@ncigdc/utils/constants.js:818]: I'm not sure if you might want the local storage variable to have the same prefix for consistency, but it shouldn't affect the deployment at all, so I don't see an issue either way.

**Review by singergr — APPROVED** [review]
Haven't tested, but it looks like it should work.

`build.sh` still unsets the old variable, but unless someone defined `REACT_APP_AWG_LOGIN_EXPIRY` before running `build.sh`, that shouldn't matter.

File: src/packages/@ncigdc/utils/constants.js (hunk 1 above)
Comment (caravinci) [inline src/packages/@ncigdc/utils/constants.js:818]:
```suggestion
  localStorage.REACT_APP_AWG_TOKEN_EXPIRY ||
```

Comment (caravinci) [reply src/packages/@ncigdc/utils/constants.js]: Thought I'd keep it without prefix for ease of use while debugging, but you're right... it should be consistent.

**Pull request merged**